tasks/sep/006-1-sc-changes/VALIDATION.md

```diff
@@ -26,31 +26,29 @@ Enables the simulation by setting the threshold to 1:
 **Before:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
 **After:** `0x0000000000000000000000000000000000000000000000000000000000000003` <br/>
 **Meaning:** The threshold has been increased to 3. The key can be validated by the location of the `threshold` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/examples/libraries/GnosisSafeStorage.sol#L14). The before value shown is 1, rather than the actual value of 2, because of the state override.
 
 **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
-**Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
-**After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+**Before:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+**After:** `0x0000000000000000000000000000000000000000000000000000000000000003` <br/>
 **Meaning:** The nonce has been increased to 1. The key can be validated by the location of the `nonce` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/examples/libraries/GnosisSafeStorage.sol#L17).
 
-The following three changes are all updates to the `modules` mapping, which is in the [1-th slot](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L10).
-
-**Key:** `0x39ead0e6be1167a8c3acf9d6eec70b5b0f4d9e3dc111ef31bd39447081125e31` <br/>
+**Key:** `0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8` <br/>
 **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
-**After:** `0x000000000000000000000000812b1fa86be61a787705c49fc0fb05ef50c8fedf` <br/>
-**Meaning:** The `DeputyGuardianModule` at `0xed12261735aD411A40Ea092FF4701a962d25cA21` is now pointing to the `LivenessModule` at [0x812b1fa86be61a787705c49fc0fb05ef50c8fedf](https://sepolia.etherscan.io/address/0x812b1fa86be61a787705c49fc0fb05ef50c8fedf).
-  This is `modules[0xed12261735aD411A40Ea092FF4701a962d25cA21]`, so the key can be
-    derived from `cast index address 0xed12261735aD411A40Ea092FF4701a962d25cA21 1`.
+**After:** `0x0000000000000000000000004416c7fe250ee49b5a3133146a0bbb8ec0c6a321` <br/>
+**Meaning:** The `LivenessGuard` address is set to [0x4416c7fe250ee49b5a3133146a0bbb8ec0c6a321](https://sepolia.etherscan.io/address/0x4416c7fe250ee49b5a3133146a0bbb8ec0c6a321). The key can be validated by the key in the [Guard Manager](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/base/GuardManager.sol#L30).
+
+The following three changes are all updates to the `modules` mapping, which is in the [1-th slot](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L10).
```

Comment:
nit, I thought the dash was a typo for a 0 1-th
```suggestion
The following three changes are all updates to the `modules` mapping, which is in [slot 1](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L10).
```

Reply:
I thought about "first slot" but that seemed confusing. I was kind of happy with `1-th`, but happy with slot 1 too.